### ISSUE TITLE
[shaders] make "glowing" effect render in vanilla instead in shaders 

### ISSUE THREAD

**Issue by mistershkolnik** [sp614x/optifine#5759]
## Description of the new feature
It is a bit annoying to not see "glowing" effect while playing servers that does it.
I'll show you a comparison between IRIS and Optifine with doing the "glowing" effect with shaders
Oprifine screenshot: https://imgur.com/a/IMumzcb
even with fast render off (which my bug report about it got closed) https://imgur.com/a/4wJ3NjY

For example, an IRIS mod fixes it: https://imgur.com/a/QwqXxTR

**Comment by KaiAF** [comment]
I looked at other issues about this, it seems like that it just depends on the shader pack if the glowing effect works.
https://github.com/sp614x/optifine/issues/5223#issuecomment-762388229 I know that it was something with internal shaders, but he said that it should work. Does this happen on every shader?

**Comment by mistershkolnik** [comment]
Yes. It happens on all shaders that doesnt make their own version of
'glowing' effect (see complementary and BSL shaders) but the only problem
there is, it misses color

чт, 10 июн. 2021 г., 20:54 Iris ***@***.***>:

**Comment by dragon3025** [comment]
This also happens with "OptiFine HD U H4" and "OptiFine HD U H5 pre4" for 1.18.1 while using internal shaders. I don't know if I should of posted that in this post or in [#6039 ](https://github.com/sp614x/optifine/issues/6039#issue-945853242)
